### PR TITLE
Add preliminary FreeBSD support and setup.py cleanup

### PR DESCRIPTION
**Pull request by sunpoet**
- Add preliminary FreeBSD support to fix moviepy on FreeBSD 
- Remove python 2.7 stuff in setup.py of imageio 2.5.0 (py3k only)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -214,7 +210,7 @@ def run(self):
     long_description=long_description.replace("__doc__", __doc__),
     platforms="any",
     provides=["imageio"],
-    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
+    python_requires=">=3.4",
```
Comment (almarklein) [inline setup.py]: Oh, I would love this change, but let's wait a few months :)

**Comment by scivision** [comment]
This may have been discussed already elsewhere, but it might be worth considering supporting only Python &ge; 3.5 to get the benefits of type hinting and pathlib built-in. This will clean up and help make the project more robust.
Python 3.4 has been EOL'd already and Python 3.5 may be EOL in late 2020.

**Comment by almarklein** [comment]
Let's do this!

**Comment by almarklein** [comment]
Also see #484

**Pull request merged**